ApplicationLibCode/Application/Tools/RiaQDateTimeTools.cpp

```diff
@@ -499,3 +499,68 @@ QList<caf::PdmOptionItemInfo> RiaQDateTimeTools::createOptionItems( const std::v
 
     return options;
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+std::set<QDateTime> RiaQDateTimeTools::createEvenlyDistributedDates( const std::vector<QDateTime>& inputDates, int numDates )
+{
+    std::set<QDateTime> outputDates;
+    if ( inputDates.empty() || numDates <= 0 )
+    {
+        return outputDates;
```

Comment:
Move `std::set<QDateTime> outputDates;` to for-loop at line 532,  `return {};` when possible